### DISCUSSION TITLE
What am I?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1116] [Galaxy Trivia] [answered]
My 12, 8, and 9 is a sheep.
My 3, 5, and 6 is not high.
My 1, 8, and 9 is a starchy vegetable.
My 7, 8, 4, and 5 is a glow around the head.
My 12, 11, 2, and 3 is used in fishing.
My 7, 2, and 10 can be found at the bottom of a skirt.
My whole is a type of bird.

**Comment by peterbaccaro** [comment]
1. Ram
2. Low
3. Yam
4. Halo
5. Reel
6. Hem

You are a **Yellowhammer**.

**Comment by thuongtruong1009** [comment, marked as answer]
You are a **partridge**. Let's break down the clues:

1. **12, 8, and 9 is a sheep:** This refers to "EWE," which is a female sheep.
2. **3, 5, and 6 is not high:** This refers to "LOW," which is the opposite of high.
3. **1, 8, and 9 is a starchy vegetable:** This refers to "PEA," which is a starchy vegetable.
4. **7, 8, 4, and 5 is a glow around the head:** This refers to "HALO," which is a glow often depicted around the head of angels or saints.
5. **12, 11, 2, and 3 is used in fishing:** This refers to "NET," which is used in fishing to catch fish.
6. **7, 2, and 10 can be found at the bottom of a skirt:** This refers to "TUTU," which is a type of skirt often worn in ballet.
7. **My whole is a type of bird:** When you put all the answers together: EWE + LOW + PEA + HALO + NET + TUTU, you get "partridge," which is a type of bird.
